src/tigerbeetle-java/src/main/java/com/tigerbeetle/Client.java

```diff
@@ -26,43 +20,103 @@ public final class Client implements AutoCloseable {
     private long packetsHead;
     private long packetsTail;
 
-    public Client(int clusterID, String[] replicaAddresses) {
-        this(clusterID, replicaAddresses, DEFAULT_MAX_CONCURRENCY);
-    }
-
+    /**
+     * Initializes an instance of TigerBeetle client. This class is thread-safe and for optimal
+     * performance, a single instance should be shared between multiple concurrent tasks.
+     * <p>
+     * Multiple clients can be instantiated in case of connecting to more than one TigerBeetle
+     * cluster.
+     *
+     * @param clusterID
+     * @param replicaAddresses
+     * @param maxConcurrency
+     *
+     * @throws InitializationException If TigerBeetle an error occurred initializing this client.
+     *         See {@link InitializationException.Status} for more details.
+     *
+     * @throws IllegalArgumentException If {@code clusterID} is negative.
```

Comment:
You've been inconsistent in this PR about capitalize `if/If` after `@throws`. I'm not sure which one it should be but they should probably all be the same?

Reply:
They should be capitalized only in top-level comments, never after tags like `@param`, `@return`, and `@throws`.